backend/canisters/translations/api/can.did

```diff
@@ -1 +1,94 @@
-service : {}
+import "../../../libraries/types/can.did";
+
+type ProposeArgs = record {
+    locale : text;
+    key : text;
+    value : text;
+};
+
+type ProposeResponse = variant {
+    Success : nat64;
+    UserNotFound;
+    InvalidArgs : text;
+    InternalError : text;
+};
+
+type Approve2Args = record {
+    id : nat64;
+};
+
+type ApproveResponse = variant {
+    Success;
+    NotProposed;
+    NotFound;
+    NotAuthorized;
+    InternalError : text;
+};
+
+type RejectArgs = record {
+    id : nat64;
+};
+
+type RejectResponse = variant {
+    Success;
+    NotProposed;
+    NotFound;
+    NotAuthorized;
+    InternalError : text;
+};
+
+type MarkDeployedArgs = record {
+    latest_approval : TimestampMillis;
+};
+
+type MarkDeployedResponse = variant {
+    Success;
+};
+
+type ProposedResponse = variant {
+    Success : ProposedSuccessResult;
+    NotAuthorized;
+    InternalError : text;
+};
+
+type ProposedSuccessResult = record {
+    records : vec Record;
+};
+
+type Record = record {
+    locale : text;
+    key : text;
+    candidates : vec CandidateTranslation;
```

Comment:
I wasn't expecting this. I thought that we only ever have one candidate translation. Last proposed wins.